tests/integration/test_rpm.py

```diff
@@ -230,6 +230,26 @@ def read_and_normalize_repofile(path: Path) -> str:
 @pytest.mark.parametrize(
     "test_params, check_cmd, expected_cmd_output",
     [
+        pytest.param(
+            utils.TestParameters(
+                branch="rpm/compress-type",
+                packages=(
+                    {
+                        "type": "rpm",
+                        "options": {
+                            "dnf": {"ubi-7": {"gpgcheck": 0}},
+                        },
+                    },
+                ),
+                flags=["--dev-package-managers"],
+                check_vendor_checksums=False,
+                expected_exit_code=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            ["vim", "--version"],
+            ["Vi IMproved 7.4"],
+            id="rpm_compress_type",
```

Comment:
nitpick: the test could be renamed to something more meaningful, see the suggestions you got in the other pr.